commands/update/nodepool/command.go

```diff
@@ -113,10 +130,23 @@ func verifyPreconditions(args Arguments) error {
 		return microerror.Mask(errors.ClusterNameOrIDMissingError)
 	} else if args.NodePoolID == "" {
 		return microerror.Mask(errors.NodePoolIDMissingError)
-	} else if args.ScalingMin == 0 && args.ScalingMax == 0 && args.Name == "" {
-		return microerror.Mask(errors.NoOpError)
-	} else if args.ScalingMin > args.ScalingMax && args.ScalingMax > 0 {
-		return microerror.Mask(errors.WorkersMinMaxInvalidError)
+	}
+
+	if args.Provider == provider.AWS {
+		if args.ScalingMin == 0 && args.ScalingMax == 0 && args.Name == "" {
+			return microerror.Maskf(errors.NoOpError, "Nothing to update.")
+		} else if args.ScalingMin > args.ScalingMax && args.ScalingMax > 0 {
+			return microerror.Mask(errors.WorkersMinMaxInvalidError)
+		}
+	}
+
+	if args.Provider == provider.Azure {
+		if args.ScalingMin > 0 || args.ScalingMax > 0 {
+			return microerror.Maskf(errors.NoOpError, "Provider '%s' does not support node pool scaling.", args.Provider)
```

Comment:
Azure does support node pool size change, but not auto-scaling. Should the check maybe be `args.ScalingMin != args.ScalingMax` ?

Reply:
I updated the code and the screenshots 👍